### ISSUE TITLE
Justification/Orientation controls: Replace implementation with ToggleGroupControl

### ISSUE THREAD

**Issue by mirka** [WordPress/gutenberg#43416]
These buttons (`.block-editor-hooks__flex-layout-justification-controls`, `.block-editor-hooks__flex-layout-orientation-controls`) should be replaced with the new icon button variant of ToggleGroupControl.

<img width="271" alt="The Justification/Orientation controls" src="https://user-images.githubusercontent.com/555336/185596714-6d423f1a-8ce0-464b-ad37-5858667077d5.png">

This will improve consistency, reduce code duplication, and add visual/semantic indications that the options are mutually exclusive.
